real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/OldLogConverter.java

```diff
@@ -0,0 +1,169 @@
+package com.roboclub.robobuggy.logging;
+import java.util.*;
+import java.io.*;
+
+
+public class OldLogConverter {
+	
+/** Steering Angle Conversion Rate */
+static private final int ARD_TO_DEG = 100;
+/** Steering Angle offset?? */
+static private final int OFFSET = -200;
+	
+static ArrayList<String> sensor_type = new ArrayList<String>();
+static ArrayList<Integer> sensor_quant = new ArrayList<Integer>();
```

Comment:
I cut down the size of my file by deleting most of the data. I think I
still have a reading from each of the sensors, but let me know if you need
anything else. (The splice is before the last sensor reading, so if
something's wrong there, that might be why.)

- David

{
    "name": "Robobuggy Data Logs",
    "schema_version": 1.0,
    "date_recorded": "10/04/2015",
    "software_version": "1.0.0",
    "sensor_data": [
        {"name":"Steering","params":{"angle": "102.0"},"timestamp":"07:15:51.737"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":-349.4399620313241,"acceleration":-386.9767021387864},"timestamp":"2015-10-04 07:15:51.737"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.737"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.737"},
        {"name":"logging button","params":{"logging_status":"start"},"timestamp":"2015-10-04 07:15:51.739"},
        {"name":"Steering","params":{"angle": "55.34"},"timestamp":"2015-10-04 07:15:51.753"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.753"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.753"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.753"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.753"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.753"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.769"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.769"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.769"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.769"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.769"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.769"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.785"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.785"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.785"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.785"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.801"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.801"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.801"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.801"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.801"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.801"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.817"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.817"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.817"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.817"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.817"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.833"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:15:51.833"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.833"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.833"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.833"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.849"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.849"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.849"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.849"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.849"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.849"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.865"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.865"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.865"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.865"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.881"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.881"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.881"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.881"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.881"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.881"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.897"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.897"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.897"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.897"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.897"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.897"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.913"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.913"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.913"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":null,"acceleration":null},"timestamp":"2015-10-04 07:15:51.913"},
        {"name":"GPS","params":{"latitude":40.433453202777436,"lat_direction":"N","longitude":79.94227549235026,"long_direction":"W","gps_quality":"1","num_satellites":"6","HDOP":2.14,"antenna_altitude":297.0},"timestamp":"2015-10-04 07:15:51.914"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.929"},
        {"name":"Encoder","params":{"dataword":0.0,"distance":0.0,"velocity":0.0,"acceleration":null},"timestamp":"2015-10-04 07:15:51.929"},
        {"name":"Steering","params":{"angle": "54.95"},"timestamp":"2015-10-04 07:15:51.929"},
        {"name":"Steering","params":{"angle": "54.55"},"timestamp":"2015-10-04 07:20:09.593"}
    ],
    "data_breakdown" : {"Steering":38502,"Encoder":38806,"logging button":2,"GPS":258,"IMU":12524}
}